vllm/engine/llm_engine.py

```diff
@@ -180,6 +181,7 @@ def __init__(
         log_stats: bool,
         usage_context: UsageContext = UsageContext.ENGINE_CONTEXT,
         stat_loggers: Optional[Dict[str, StatLoggerBase]] = None,
+        input_registry: InputRegistry = INPUT_REGISTRY,
```

Comment:
Why do we need to make this an variable of `__init__`?

Reply:
Compared to assigning the global `INPUT_REGISTRY` directly to the instance attribute, this makes it easier to see the dependencies of `LLMEngine`.